package.json

```
@@ -26,11 +26,12 @@
     "build": "yarn workspaces foreach --parallel --topological --verbose run build",
     "build:dapp": "yarn workspace @metamask/snap-simple-keyring-site build",
     "build:snap": "yarn workspace @metamask/snap-simple-keyring-snap build",
-    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:deps",
+    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:deps && yarn lint:types",
```

We should discuss whether we actually want this line to be

```suggestion
    "lint": "yarn workspaces foreach --parallel run lint",
```

Right now, we're running a weird hybrid lint, where `lint:eslint` and `lint:misc` are running at the root of the repo, but `lint:deps` and `lint:types` are running in each workspace.

---

Good catch! I replaced the root targets by `workspaces foreach` ones.